integration-cli/docker_api_build_test.go

```diff
@@ -406,6 +407,10 @@ func (s *DockerSuite) TestBuildAddRemoteNoDecompress(c *check.C) {
 }
 
 func (s *DockerSuite) TestBuildChownOnCopy(c *check.C) {
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.31") {
```

Comment:
could be a onliner : 
```go
skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.31"), "new feature added in 1.31, go version `s", testEnv.DaemonAPIVersion())
```

(not even sure we need to print the current version)

Reply:
oh right but it's with `check` so might not work 😭

Reply:
@vdemeester bingo. skip.If won't work here. They use two different interfaces.

Reply:
@AntaresS yep sorry about that, was thinking it was on `integration` tests :stuck_out_tongue: